### PR TITLE
Fix #180639 boykot.co

### LINKED ISSUE #180639: www.boykot.co
### Issue URL (Ads)
 [https://www.boykot.co/d/unilever](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fwww.boykot.co%2Fd%2Funilever)
Username: `@gototheskinny`

### Screenshots
<details>
  <summary>Screenshot 1 </summary>

  Screenshot 1
</details>

### System configuration

Information | Value
--- | ---
AdGuard product: | AdGuard for iOS v4.5.3
System version: | ios 17.3.1
License type: | paid
Advanced protection: | enabled
DNS filtering: | disabled
Filters: | <b>Ad Blocking:</b><br/>AdGuard Base, <br/>AdGuard Mobile Ads<br/><br/><b>Privacy:</b><br/>AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/>AdGuard Social Media<br/><br/><b>Annoyances:</b><br/>AdGuard Annoyances<br/><br/><b>Security:</b><br/>Phishing URL Blocklist, <br/>Scam Blocklist by DurableNapkin, <br/>uBlock Origin – Badware risks<br/><br/><b>Language-specific:</b><br/>AdGuard Turkish

### PR DESCRIPTION
**Pull request by grumaxxx**
Fix #180639

### CODE REVIEW

**Comment by grumaxxx** [comment]
I'm not sure if we should block the google play button?

**Comment by AdamWr** [comment]
> I'm not sure if we should block the google play button?

It's in the footer (not annoying), so in my opinion it's not necessary to block it.

**Review by piquark6046 — CHANGES_REQUESTED** [review]
Social sharing button: `.btn.btn-sm.btn-primary`

File: TurkishFilter/sections/specific.txt [hunk 1]
```diff
@@ -68,6 +68,8 @@ $xmlhttprequest,third-party,denyallow=rsc.cdn77.org|linkvertise.com|fastly.net|s
 /service/advertchannels?$xmlhttprequest,redirect=nooptext,domain=mardinhaber.com.tr|gidahatti.com|bolugundem.com
 dirilispostasi.com,turkuazgazetesi.net,yenialanya.com,bugunkocaeli.com.tr,cagdaskocaeli.com.tr,mansetaydin.com,seskocaeli.com,kocaelibarisgazetesi.com,kocaelikoz.com,enkocaeli.com##.block-pano > div.us
 !
+boykot.co###Muslimer
+boykot.co##.btn.btn-sm.btn-primary, .btn.btn-sm.btn-dark
```
Comment (piquark6046) [inline TurkishFilter/sections/specific.txt:72]: As far as i know, multiple selectors are not allowed.

**Review by piquark6046 — CHANGES_REQUESTED** [review]

**Review by AdamWr — APPROVED** [review]

**Review by piquark6046 — APPROVED** [review]

**Pull request merged**